sphinx/builders/__init__.py

```diff
@@ -52,6 +55,13 @@
 logger = logging.getLogger(__name__)
 
 
+CONFIG_CHANGED_REASON = {
```

Comment:
nits: Why is this REASON defined here instead of `environment/__init__.py`?

Reply:
These message catalog is only used here. Either is fine to me.

Reply:
I like putting it in `environment/__ init__.py`, but I do not have a strong opinion.

Reply:
Acceptable. done